frontend/wallet/src/render/components/TransactionsView.re

```diff
@@ -92,14 +110,35 @@ let extractTransactions: Js.t('a) => array(TransactionCell.Transaction.t) =
 
 [@react.component]
 let make = () => {
-  let transactionQuery = Transactions.make(~publicKey="123", ());
+  let activeWallet = Hooks.useActiveWallet();
+  let activeWalletKey =
+    Option.map(~f=PublicKey.toString, activeWallet)
+    |> Option.withDefault(~default="");
+  let transactionQuery = Transactions.make(~publicKey="123", ~after="", ());
```

Comment:
Should we update this to be the active wallet?

Reply:
Yeah i'll do this in the pending txn pr